From ba720e8f2c3ca3b90d6486fe97d8df0c82e6ec50 Mon Sep 17 00:00:00 2001
From: Richard Zhao <richard.zhao@freescale.com>
Date: Thu, 7 Jul 2011 15:42:40 +0800
Subject: [PATCH 0214/2463] ENGR00152773 imx6: clk: *_axi_clk get_rate should
 be NULL

Clocks that can not change rate should set get_rate to NULL.

Signed-off-by: Richard Zhao <richard.zhao@freescale.com>
---
 arch/arm/mach-mx6/clock.c |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/arch/arm/mach-mx6/clock.c b/arch/arm/mach-mx6/clock.c
index 895dd9f..db20143 100644
--- a/arch/arm/mach-mx6/clock.c
+++ b/arch/arm/mach-mx6/clock.c
@@ -1257,7 +1257,6 @@ static struct clk gpu2d_axi_clk = {
 	__INIT_CLK_DEBUG(gpu2d_axi_clk)
 	.parent = &axi_clk,
 	.set_parent = _clk_gpu2d_axi_set_parent,
-	.get_rate = _clk_axi_get_rate,
 };
 
 static int _clk_gpu3d_axi_set_parent(struct clk *clk, struct clk *parent)
@@ -1276,7 +1275,6 @@ static struct clk gpu3d_axi_clk = {
 	__INIT_CLK_DEBUG(gpu3d_axi_clk)
 	.parent = &axi_clk,
 	.set_parent = _clk_gpu3d_axi_set_parent,
-	.get_rate = _clk_axi_get_rate,
 };
 
 static int _clk_pcie_axi_set_parent(struct clk *clk, struct clk *parent)
@@ -1295,7 +1293,6 @@ static struct clk pcie_axi_clk = {
 	__INIT_CLK_DEBUG(pcie_axi_clk)
 	.parent = &axi_clk,
 	.set_parent = _clk_pcie_axi_set_parent,
-	.get_rate = _clk_axi_get_rate,
 };
 
 static int _clk_vdo_axi_set_parent(struct clk *clk, struct clk *parent)
@@ -1318,7 +1315,6 @@ static struct clk vdo_axi_clk = {
 	.enable = _clk_enable,
 	.disable = _clk_disable,
 	.set_parent = _clk_vdo_axi_set_parent,
-	.get_rate = _clk_axi_get_rate,
 };
 
 static struct clk vdoa_clk = {
-- 
1.7.7.4

